src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -86,6 +86,13 @@ public int GetArrayLength()
             return _parent.GetArrayLength(_idx);
         }
 
+        internal int GetObjectSize()
```

Comment:
@bartonjs I think that might be a useful addition to the type's public API.

Reply:
How is size defined?

Reply:
I suppose we could also call it `GetObjectCount()` reflecting the fact that objects are essentially dictionaries that do use the term "count".

Reply:
`GetPropertyCount()` sounds more appropriate, since what you're counting are properties.

Sounds like a good API proposal for .NET 10.

Reply:
https://github.com/dotnet/runtime/issues/created_by/@me